content/en/docs/hertz/tutorials/framework-exten/log.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Logger Extension"
+linkTitle: "Logger Extension"
+date: 2022-06-22
+weight: 2
+description: >
+
+---
+
+Hertz provides logger extension, and the interface is defined in `pkg/common/hlog`. 
+
+## Interface Defination
+
+In Hertz, the interfaces `Logger`、`CtxLogger`、`FormatLogger` are defined in `pkg/common/hlog`, and these interfaces are used to output logs in different ways, and a Control interface is defined to control the logger.
```

Comment:
`Logger`, `CtxLogger` and `FormatLogger`

-> use , instead of 、

Reply:
resolved